README.md

```diff
@@ -167,7 +168,6 @@ To learn about a specific command, just include the name of the command (For exa
     * `AZURE_STORAGE_SAS_TOKEN`: Specifies the SAS token to use for authentication.
 - Managed Identity auth:
     * `AZURE_STORAGE_IDENTITY_CLIENT_ID`: Only one of these three parameters are needed if multiple identities are present on the system.
-    * `AZURE_STORAGE_IDENTITY_OBJECT_ID`: Only one of these three parameters are needed if multiple identities are present on the system.

```

Comment:
In AzStorage can we add an error message and fail the mount saying object-id is no longer supported kindly use azcli or use app-id based authentication. Otherwise there will be issues coming in where customers will say their mount failed and we need to debug why it failed. Better to give upfront error message stating the same.

Reply:
Same as previous, objectID is not deprecated, but has dependency on Azure CLI being installed.

Reply:
We shall also mark this PR and store it somewhere, when and if AzIdentity adds the support we will have to revert this exact change.